core/Mutations/DifferentialEvolutionMutation.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+from core import Mutation, Individual, Population
```

Comment:
this should be changed to `from core.Mutation import Mutation` etc. since in your case you import a module, not a class

Reply:
done